client/signup/steps/plans/tabbed-plans.js

```diff
@@ -0,0 +1,1316 @@
+								key={ `planHeader${ index }` }
+								className={ `tabbed-plans__header-${ index + 1 }` }
+							>
+								{ item.planName }
+								{ item.planName === 'Personal' && <span>Best to start</span> }
+								{ item.planName === 'Business' && <span>Best value</span> }
+								<p>{ bestForStrings[ item.planName ] }</p>
+							</PlanHeader>
+							<PlanPrice
+								key={ `planPrice${ index }` }
+								className={ `tabbed-plans__price-${ index + 1 }` }
+							>
+								{ formatCurrency( item.rawPrice, item.currencyCode, { stripZeros: true } ) }
+							</PlanPrice>
+							<TermDescription className={ `tabbed-plans__term-desc-${ index + 1 }` }>
+								{ item.planName === 'Free' && <span>Limited features</span> }
+								{ item.termLength === 'annually' && item.planName !== 'Free' && (
+									<>
+										<span>per month, billed annually</span>
+										<Savings>
+											{ "You're saving " +
+												Math.round(
+													( ( item.rawPriceForMonthly - item.rawPrice ) /
+														item.rawPriceForMonthly ) *
+														100
+												) +
+												'% by paying annually' }
+										</Savings>
+									</>
+								) }
+								{ termLength === 'monthly' && <span>per month, billed monthly</span> }
+							</TermDescription>
+							{ item.planName !== 'Free' && (
+								<CtaButton
+									key={ `planCtaTop${ index }` }
+									className={ `tabbed-plans__top-button-${ index + 1 }` }
+									onClick={ () => handleUpgradeButtonClick( item.planSlug, item.planProductId ) }
+									primary={ item.planName === primaryButton }
+								>
+									{ `Start with ${ item.planName }` }
+								</CtaButton>
+							) }
+							<CtaButton
+								key={ `planCta${ index }` }
+								className={ `tabbed-plans__button-${ index + 1 }` }
+								onClick={ () => handleUpgradeButtonClick( item.planSlug, item.planProductId ) }
+								primary={ item.planName === primaryButton }
+							>
+								{ `Start with ${ item.planName }` }
+							</CtaButton>
+							<SharedFeatures
+								selectedTab={ selectedTab }
+								sharedFeatures={ sharedFeatures }
+								showFeatures={ showFeatures }
+								setShowFeatures={ setShowFeatures }
+								className={ classNames(
+									'tabbed-plans__shared-features-mobile',
+									`tabbed-plans__shared-features-${ index + 1 }`
+								) }
+							/>
+						</React.Fragment>
+					) ) }
+
+				{ featureComparison.map( ( item, index, arr ) => (
+					<React.Fragment key={ `feature${ index }` }>
+						<FeatureTitle
+							key={ `featureTitle${ index }` }
+							className={ `tabbed-plans__feature-title-${ index + 1 }` }
+							isLast={ arr.length === index + 1 }
+						>
+							{ item.featureName }
+							<InfoPopover
+								className="tabbed-plans__feature-tip-info"
+								position="right"
+								icon="info"
+								iconSize={ 15 }
+							>
+								{ item.tooltip }
+							</InfoPopover>
+						</FeatureTitle>
+						{ item.planOne && (
```

Comment:
The following three blocks of code, `{ item.planOne && ...`, `{item.planTwo && `, `{item.planThree && ...` are basically the same, with the only difference being `planOne`, `planTwo` or `planThree`. Let's DRY them here :)

Reply:
Yes, this section isn't great, there's also some CSS shenanigans happening. If you don't mind, and in the interest of timing, I suggest launching this version as-is, as ugly as it is, because this section in particular will benefit from the refactor I'm working on.